eclipse-collections/src/main/java/org/eclipse/collections/impl/map/mutable/UnifiedMap.java

```diff
@@ -2145,6 +2149,26 @@ public boolean isEmpty()
             return UnifiedMap.this.isEmpty();
         }
 
+        @Override
+        public K getFirst()
+        {
+            if (UnifiedMap.this.size() > 0)
+            {
+                return (K) this.toArray()[0];
```

Comment:
@johnjstuart Just use `this.iterator().next()`?

Reply:
@johnjstuart or `IterableIterate.getFirst` as in `getLast`? It's really a pity to convert the iterable to an array just for this